### PR TITLE
Clean up additional registry branches when uninstalling Python on Windows

### PR DESCRIPTION
**Pull request by maxim-lobanov**
1) Clean up `Uninstall` section in registry
2) Remove x86 version of tool when install x64
3) Check registry always

### CODE REVIEW

File: installers/win-setup-template.ps1 [hunk 1]
```diff
@@ -13,15 +12,27 @@ function Remove-RegistryEntries
...
+    (
+        [Parameter(Mandatory)][String] $Architecture,
+        [Parameter(Mandatory)][Int32] $MajorVersion,
+        [Parameter(Mandatory)][Int32] $MinorVersion
+    )
+
+    $versionFilter = Get-RegistryVersionFilter -Architecture $Architecture -MajorVersion $MajorVersion -MinorVersion $MinorVersion
+
     $regPath = "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Installer\UserData\S-1-5-18\Products"
     $regKeys = Get-ChildItem -Path Registry::$regPath -Recurse | Where-Object Property -Ccontains DisplayName
```
Comment (vsafonkin) [inline installers/win-setup-template.ps1:37]: `-Ccontains` - typo?

Comment (vsafonkin) [inline installers/win-setup-template.ps1:37]: `-Ccontains` - typo?

Comment (maxim-lobanov) [reply installers/win-setup-template.ps1:37]: https://ss64.com/ps/ccontains.html

**Pull request merged**